### PR TITLE
ci: optimise when to run in the jjbb

### PR DESCRIPTION
**Pull request by v1v**

### CODE REVIEW

**Comment by devcorpio** [comment]
Hi @v1v 

Do you know why the builds are failing? (The same thing is happening in my PR). 

I'm seeing something about "Unable to find a build for artifact copy".

File: .ci/jobs/defaults.yml [hunk 1]
```diff
@@ -33,11 +33,14 @@
           credentials: f6c7695a-671e-4f4f-a331-acdce44ff9ba
         build-strategies:
         - tags:
-            ignore-tags-older-than: -1
+            ignore-tags-older-than: 30
             ignore-tags-newer-than: -1
```
Comment (kuisathaverat) [inline .ci/jobs/defaults.yml:37]: the options are inverted is a bug

**Comment by v1v** [comment]
> Do you know why the builds are failing? (The same thing is happening in my PR).

I don't have enough context but there was some issue with the GitHub API quota today and several builds were stalled. I bet that's the issue about, or one of the reasons.

File: .ci/jobs/defaults.yml (hunk 1 above)
Comment (v1v) [inline .ci/jobs/defaults.yml:37]:
```suggestion
            ignore-tags-older-than: -1
            ignore-tags-newer-than: 30
```

**Review by kuisathaverat — APPROVED** [review]

**Review by cachedout — APPROVED** [review]

**Pull request merged**